tests/utils/test_sampling_utils.py

```diff
@@ -28,3 +30,40 @@ def test_sample_points_on_plane() -> None:
     np.testing.assert_almost_equal(pts_residuals, np.zeros((num_points, 1)))
 
     assert pts.shape == (10, 3)
+
+
+def test_sample_random_directions() -> None:
+    num_samples = 1000
+
+    direction_list = sampling_utils.sample_random_directions(num_samples)
+    assert len(direction_list) == num_samples
+
+    directions = np.array([direction.point3() for direction in direction_list])
+    mean_direction = np.mean(directions, axis=0)
+    std_direction = np.std(directions, axis=0)
+    std_direction_relative = std_direction / std_direction[0]
+
+    # Increase number of samples for a smaller threshold
+    assert np.allclose(mean_direction, np.zeros(3), atol=1e-1)
+    assert np.allclose(std_direction_relative, np.ones(3), atol=1e-1)
+
+
+def test_sample_kde_directions() -> None:
```

Comment:
nit: could we add a brief 1-sentence docstring to document in prose what we are testing here?